doc/basic_configuration.rst

```diff
@@ -17,10 +17,17 @@ configuration to generate the necessary Jobs on the Jenkins master.
 Create / fork configuration repository
 --------------------------------------
 
-First you need to either fork the
+First you need to either fork or clone the
```

Comment:
The note you mentioned is the main reason why simply cloning the repo to your local file system isn't sufficient. Therefore I think this should stay "fork"-only.

Reply:
It doesn't need to be a fork though, it doesn't even need to be a clone. A normal download is sufficient if even can be on the same machine. The only requirement is that it is accessible through http, since file system references won't work.

Reply:
That is true. The documentation just describes the most common (and easy) use case. The user is always free to diverge from that recommendation. Setting up a HTTP server to serve the files isn't something I think this documentation should describe.

Reply:
Thanks! Updated. I hope it's clear enough now.